ui/README.md

```
@@ -0,0 +1,54 @@
+### Build for Production
+
+```bash
+npm run build
+npm run preview
+```
+
+The build output is emitted to `dist/` and can be served from AWS S3 + CloudFront, an App Runner service, or containerized alongside backend APIs.
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)